main.css

```diff
@@ -419,4 +419,40 @@ input[type="radio"] {
     width: var(--canvas-cell-width); 
     height: var(--canvas-cell-width);
     opacity: 0.5; 
+}
+
+/* buy me a coffee support section */
+#coffee-support {
+	background-color: var(--blanket-background-color);
```

Comment:
The only requests I have are:
- can you do `background-color: var(--body-background-color)`
- can you remove the `box-shadow`?

These changes are motivated by make the card more subtle. I don't want to feel like the user is being advertised to, if you know what you mean. Subtle is fine.

Reply:
@badembafr Perfect, thank you for doing this! Nothing more, we can merge this!